utils/migration_tools/model_registry_to_runs/README.md

```diff
@@ -0,0 +1,76 @@
+
+## Why Migrate?
+Our revamped Web App adds a host of new features to Experiments, including better grouping, and enhanced reporting capabilities. By migrating your models and model versions to Experiments, you can take advantage of these features to better organize and compare your models.
+
+As a part of our focus on performance this year, the `Run` API is also be highly optimized for speed. Using the `Run` object instead of `Model` or `ModelVersion` objects will allow you to take advantage of these optimizations.
+
+This script leverages the new *Group Tags* to organise your models and model versions in Experiments, and *tags* to create saved views separating model metadata runs from training runs. This lets you view all Models and Model Versions in the same table as shown below:
+
+![Organise Models in Experiments](https://neptune.ai/wp-content/uploads/2024/07/MRtoRun.png)
+
+In this screenshot, _Group Tag_, _Custom Run ID_, and _ID_ correspond to the original Model ID, Model Version ID and the new Run ID respectively.
+
+Having all model metadata in Experiments also lets you use Experiment's native [comparisons](https://docs.neptune.ai/usage/tutorial/#compare-the-runs) and [Reports](https://docs.neptune.ai/app/reports/) to compare models and model versions.
+
+
+## Prerequisites
+
+Before using this script, make sure you have the Neptune environment variables set up. For instructions, see the [documentation](https://docs.neptune.ai/setup/setting_credentials/).
+
+Additionally, ensure that the project you want to copy metadata to has already been created.
+
+## Instructions
+
+To use the script, follow these steps:
+
+1. Execute `model_to_run.py`.
+2. Enter the project name you want to copy the model metadata from, in the `WORKSPACE_NAME/PROJECT_NAME` format. Leave this prompt as black to use the `NEPTUNE_PROJECT` environment variable.
+3. Enter the number of workers to use to copy the metadata. Leave blank to use all available CPUs.
+4. The script will generate run logs in the same folder as `model_to_run.py`. You can modify this location by editing the `logging.basicConfig()` function.
+
+
+## Note
+
+There are a few things to keep in mind when using this script:
+
+- All models and model versions will be copied. It is not possible to filter them currently†.
+- Most of the namespaces from the model/model_versions will be retained in the runs, except for the following:
+  - `sys` namespace:
+    - The `state` field cannot be copied.
+    - The `description`, `name`, and `tags` fields are copied to the `sys` namespace in the new run.
+    - All other fields are copied to a new `old_sys` namespace in the new run.
+- The _Model Stage_ is currently copied to `old_sys/stage` field. Unlike the `sys/stage` field, this field cannot be updated from the Web App. If you want to be able to update the _Model Stage_ from the Web App, the script can be modified to copy the stage as _Tags_ instead. 
```

Comment:
```suggestion
- The _Model Stage_ is currently copied to `old_sys/stage` field. Unlike the `sys/stage` field, this field cannot be updated from the web app. To be able to update the _Model Stage_ from the web app, modify the script to copy the stage as _Tags_ instead. 
```

Reply:
Phrased as _the script can be modified_ instead of _modify the script_ as we we would do the modification and not expect the user to do it from their end